Week1/homework/app.js

```diff
@@ -1,11 +1,178 @@
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
-
-  // Replace with your own code
-  console.log(bookTitles);
+// *** 1.1
+const bookTitles = [
+  'the_brothers_karamazov',
+  'jane_eyre',
+  'to_kill_a_mockingbird',
+  'war_and_peace',
+  'don_quixote',
+  'animal_farm',
+  '1984',
+  'the_little_prince',
+  'frankenstein',
+  'crime_and_punishment',
+];
+// *** 1.2
+// console.log(bookTitles);
+
+// *** 1.3
+function list() {
+  for (let i = 0; i < bookTitles.length; i++) {
+    const listItem = document.createElement('li');
+    listItem.innerHTML = bookTitles[i];
+    createList.appendChild(listItem);
+  }
+}
+const createDiv = document.createElement('div');
+createDiv.id = 'bookList';
+document.getElementsByTagName('body')[0].appendChild(createDiv);
+
+const createList = document.createElement('ul');
+document.getElementById('bookList').appendChild(createList);
```

Comment:
This works, but you could also do `createDiv.appendChild(createList);`

Reply:
code changed